source/reference/template-apps.txt

```diff
@@ -275,7 +278,7 @@ Template Apps Available
        :ref:`Atlas GraphQL API <graphql-api>`.
      - React/ GraphQL
 
-   * - ``xamarin.todo.flex``
-     - Xamarin + Atlas Device Sync Starter
+   * - ``maui.todo.flex``
```

Comment:
Same thing here - we should move the Maui app up in the list to the appropriate spot, which I guess would be around ln 250?